tests/test_ext_imgconverter.py

```diff
@@ -1,4 +1,9 @@
-"""Test sphinx.ext.imgconverter extension."""
+"""Test sphinx.ext.imgconverter extension.
+
+Unix only. Fails if ImageMagick_ is not installed.
```

Comment:
I believe https://github.com/sphinx-doc/sphinx/pull/10411 will be able to remove this comment.